### ISSUE TITLE
Cassandra update existing entity - adding index issue

### ISSUE THREAD

**Issue by jossef** [Impetus/kundera#589]
Hey,

I'm using the latest release (2.11) with Cassandra
I have this line in my persistence.xml [taken from wiki](https://github.com/impetus-opensource/Kundera/wiki/Schema-Generation):

```
<property name="kundera.ddl.auto.prepare" value="update" />
```

This is my existing entity:

```
@Entity
@Table(name = "alerts")
@IndexCollection(columns = { @Index(name = "alertState"))
public class Alert
    ...
```

I modified it by adding a new field to be indexed:

```
@Entity
@Table(name = "alerts")
@IndexCollection(columns = { @Index(name = "alertState"), @Index(name = "date") })
public class Alert
    ...
```

Afterwards, i executed my code again and noticed that the change iv'e made has not been detected by the update component and my new index was not created.

**Comment by kkmishra** [comment]
Hi,

Confirming its an issue with thrift base schema generation, however it should work with cql3.
To use cql3 pass property map while creating EntityManagerFactory like :

```
 Map<String, String> propertyMap = new HashMap<String, String>();   
 propertyMap.put(CassandraConstants.CQL_VERSION, CassandraConstants.CQL_VERSION_3_0);

 EntityManagerFactory emf = Persistence.createEntityManagerFactory("pu-name", propertyMap);
```

a fix will be added in current trunk branch.

Thanks
Kuldeep

**Comment by chhavigangwal** [comment]
Fixed with Kundera-2.12. Please verify.

Chhavi
